controllers/provisioning_controller.go

```diff
@@ -40,6 +40,8 @@ const (
 	ComponentName = "cluster-baremetal-operator"
 	// BaremetalProvisioningCR is the name of the provisioning resource
 	BaremetalProvisioningCR = "provisioning-configuration"
+	// ContainerImagesFile volume mounted file containing the images configmap
+	ContainerImagesFile = "/etc/cluster-baremetal-operator-config/images/images.json"
```

Comment:
cluster-baremetal-operator-config is the name of the configmap. I'll change the name here and rename the configmap. I want to stay away from naming all  our resources cluster-baremetal-operator just so it might be easier to debug any issues. How does cbo-config for the configmap sound?

Reply:
Not sure whether this is fixed. Its still showing up as /etc/cluster-baremetal-operator-config here whereas the manifests show /etc/cluster-baremetal-operator.

Reply:
Fixed this discrepancy.